### PR TITLE
Create server with NodeJs and express 

### PR DESCRIPTION
**Pull request by tamerjb**

### CODE REVIEW

File: packages/server/index.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+const express = require('express');
+const mongoose = require('mongoose');
+const app = express();
+require('dotenv').config();
+
+const { MONGODB_URI = 'mongodb://localhost:27017/tictac' } = process.env;
```
Comment (yonatanAiz) [inline packages/server/index.js]: Please destructure both MONGODB_URI and PORT from same line

```suggestion
const { MONGODB_URI = 'mongodb://localhost:27017/tictac', PORT = 3000 } = process.env;
```

File: packages/server/index.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+mongoose.connect(MONGODB_URI);
+
+
+
+const { PORT = 3000 } = process.env;
+app.use(express.json());
+
+    app.listen(PORT, () => {
+      console.log(`App listening on port ${PORT}`);
+    });
```
Comment (yonatanAiz) [inline packages/server/index.js:13]: Fix indentation

**Review by yonatanAiz — CHANGES_REQUESTED** [review]
Please delete autogenerated files, DS_Store

**Review by yonatanAiz — APPROVED** [review]

**Review by Chen-Abudi — APPROVED** [review]
Nice thanks !

**Pull request merged**